Modules/Loadable/Markups/MRML/vtkMRMLMarkupsCurveNode.cxx

```diff
@@ -35,10 +37,19 @@
 #include <vtkPointData.h>
 #include <vtkPolyData.h>
 #include <vtkStringArray.h>
+#include <vtkOBBTree.h>
+#include <vtkPointLocator.h>
+#include <vtkPolyDataNormals.h>
+#include <vtkDoubleArray.h>
```

Comment:
Double-check that all includes are required and insert them in alphabetical order (it helps in avoiding including the same include multiple times)